app/index.js

```diff
@@ -1 +1,9 @@
 import 'whatwg-fetch';
+import './data.json';
+import './twitch/add-games-to-list';
+
+const ul = document.querySelector('.results');
+// Look up a `ul` element with a class called `results`
+const results = document.createElement('div');
```

Comment:
You shouldn't need to create `div.results` if it already exists in the starting HTML.